main.py

```diff
@@ -69,8 +71,8 @@ async def redirect():
 
 # Mount the output files from tools
 app.mount(
-    "/api/files/tool-output",
-    StaticFiles(directory="tool-output", check_dir=False),
+    "/api/files/output",
```

Comment:
did this also work with subfolders? when I tested it, it didn't work, that's why I did this in CL:

```
def mount_static_files(directory, path):
    if os.path.exists(directory):
        for dir, _, _ in os.walk(directory):
            relative_path = os.path.relpath(dir, directory)
            mount_path = path if relative_path == "." else f"{path}/{relative_path}"
            logger.info(f"Mounting static files '{dir}' at {mount_path}")
            app.mount(mount_path, StaticFiles(directory=dir), name=f"{dir}-static")


# Mount the data files to serve the file viewer
mount_static_files("data", "/api/files/data")
# Mount the output files from tools
mount_static_files("output", "/api/files/output")
```